components/glean/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="mozilla.telemetry.glean" />
```

Comment:
This is important, it means that our re-packaged `glean` package will export the same Kotlin package as the standalone Glean.